Athena setup steps (#2980)
quiltdata/quilt: docs/advanced-features/athena.md

Users can write SQL queries to select packages (or files from within packages)
using predicates based on package or object-level metadata.

## Prerequisites: Athena setup

To get enable Athena, you need to set up a role with policy allowing Athena.
Steps required to do this:

> 1. Create Athena policy. Go to console.aws.amazon.com/iam, and create new policy with this JSON.

Reviewer: Why not `console.aws.amazon.com/iam` clickable?

> ```

Reviewer: ```suggestion
```json
```

> {

Reviewer: Is this `arn:aws:iam::aws:policy/AmazonAthenaFullAccess`? If so I don't think there is sense to copy it here because it's AWS managed and already exists in every account. Also I think it provides more permissions than we need, so I think it either should be restricted or we should say it might be too permissive.

            "Action": [
                "s3:GetObject",
                "s3:ListBucket"
            ],
            "Resource": [
                "arn:aws:s3:::athena-examples*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:ListBucket",
                "s3:GetBucketLocation",
                "s3:ListAllMyBuckets"
            ],
            "Resource": [
                "*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "sns:ListTopics",
                "sns:GetTopicAttributes"
            ],
            "Resource": [
                "*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "cloudwatch:PutMetricAlarm",
                "cloudwatch:DescribeAlarms",
                "cloudwatch:DeleteAlarms"
            ],
            "Resource": [
                "*"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "lakeformation:GetDataAccess"
            ],
            "Resource": [
                "*"
            ]
        }
    ]
}
```

> 2. Attach ARN of this policy to your stack's template in CloudFormation. Go to CloudFormation, click "Update", and fill ARN address to "ManagedUserRoleExtraPolicies" field

Reviewer: Consider this:
```suggestion
2. Attach this policy to your CloudFormation stack. Go to CloudFormation console, select stack, click "Update", and fill ARN to "ManagedUserRoleExtraPolicies" field
```

3. Add "un-managed" Athena policy to Quilt catalog. Go to http://your-quilt-stack/admin, scroll to "Policies", click on "+" button. Click "Manually set ARN" and enter ARN of Athena policy

> 4. Attach policy to an existing Quilt role, or create a new role and attach policy to it.

Reviewer: I think we should be consistent in having/not having point at the end of list items.
